src/view/explorer/SceneList.svelte

```diff
@@ -230,9 +230,9 @@
   }
 
   function doWithUnknown(fileName: string, action: "add" | "ignore") {
-    const currentDraftIndex = $drafts.findIndex(
-      (d) => d.vaultPath === $selectedDraft.vaultPath
-    );
+    const currentDraftIndex =
+      $selectedDraft &&
+      $drafts.findIndex((d) => d.vaultPath === $selectedDraft.vaultPath);
```

Comment:
I think this would be more readable (and not make people reason about an undefined `currentDraftIndex` as

```suggestion
    if (!$selectedDraft) {
        return
    }
    const currentDraftIndex = $drafts.findIndex((d) => d.vaultPath === $selectedDraft.vaultPath);
```